css/encore.css

```diff
@@ -1 +1,109 @@
 @import url("./style.css");
+
+.grid-wrapper {
+  display: grid;
+  height: 100vh;
+  grid-template-columns: 23rem 1fr;
+  /* grid-template-rows: 70spx auto; */
+  grid-template-areas:
+    "header main"
+    "header main";
+}
+
+.components {
+  grid-area: header;
+  padding: 1rem 2rem;
+  background-color: var(--color-secondary);
+  box-shadow: 0px 2px 10px 2px var(--color-primary);
+  color: var(--txt-light);
+  font-size: 3rem;
+  font-weight: bold;
+}
+.components .left-nav {
+  /* width: 25%;
+  height: calc(100vh);
+  position: fixed;
+  top: 0;
+  left: 0; */
+}
+
+.components h1 a {
+  transition: 0.5s;
+  text-decoration: none;
+  padding: 1%;
+  color: #fff;
+  font-weight: 800;
+  font-size: 5rem;
+}
+.components h1 a:hover {
+  /* content: "Click Me!"; */
```

Comment:
You can remove the unwanted codes ( i.e: commented code )

Reply:
Done